SQLToolsConnections.sublime-settings

```diff
@@ -28,18 +28,20 @@
       "port"    :  5432,
       "database": "dbname",
       "username": "anotheruser",
-      // for PostgreSQL "password" is optional (setup "pgpass.conf" file instead)
+      // password is optional (setup "pgpass.conf" file instead)
       "password": "password",
       "encoding": "utf-8"
     },
     "Connection Oracle": {
       "type"    : "oracle",
       "host"    : "127.0.0.1",
-      "port"    :  1522,
+      "port"    :  1521,
       "database": "dbname",
       "username": "anotheruser",
       "password": "password",
       "service" : "servicename",
+      // nls_lang is optional
```

Comment:
We should also add it to the docs later.

Everything seems to be good!

Awesome work @tkopets!

Thanks again!

Reply:
I will work on changes to the docs in next few days.
Do you prefer those changes to docs be as part of this PR or as a new one?

Reply:
No, you are right. Better keep it. I didn't notice it was a settings file

Reply:
Actually, I cannot update the docs in this pull request as our docs are located in separate `gh-pages` branch. 😃